README.md

```diff
@@ -337,6 +338,7 @@ ENV PHP_FPM_PM_MIN_SPARE_SERVERS="20"
 ENV PHP_FPM_PM_MAX_SPARE_SERVERS="40" 
 ENV PHP_FPM_PM_PROCESS_IDLE_TIMEOUT="35"
 ENV PHP_FPM_PM_MAX_REQUESTS="500"
+ENV PHP_FPM_ACCESS_FORMAT={"C":"%C","d":"%d","f":"%f","l":"%l","m":"%m","M":"%M","n":"%n","P":"%P","p":"%p","q":"%q","Q":"%Q","r":"%r","R":"%R","s":"%s","T":"%T","t":"%t","u":"%u"}
```

Comment:
Will this escape any quotes in the values?

Reply:
Looking at https://github.com/php/php-src/blob/c3cf01b553b1a358531d0fa56e3caba51f9a3e33/sapi/fpm/fpm/fpm_log.c#L406 possible but my C is a bit Rusty but I'll have a Go at it 👍

Reply:
@WyriHaximus Can we add a test for it? Is this possible?

Reply:
@carusogabriel everything is possible, it's just a matter of how much time to toss at it.

Reply:
Since exceptions are full of quotes, I'd say it's a reasonable thing to test for.

Reply:
@frankkoornstra not really for that reason, this only changes the access log format, our exceptions are already JSON. This will make 99.999999% of the log lines in JSON possible. The only two missing are the two start up lines when FPM starts

Reply:
Oh ok, in that case I'm not too worried :D